segments/tasks.py

```diff
@@ -28,7 +29,7 @@ def refresh_segment(segment_id):
         logger.exception("SEGMENTS: Unable to refresh segment id %s. DoesNotExist.", segment_id)
 
 
-@task(queue=SEGMENTS_CELERY_QUEUE)
+@shared_task(queue=SEGMENTS_CELERY_QUEUE)
```

Comment:
Why this change?

Reply:
[That is the answer](https://stackoverflow.com/a/42402387)
Shortly - we should use shared_task for dedicated libraries like this, so we don't have to create celery app instance for them. With shared task it will use celery app instance from main repo